src/components/Form.jsx

```diff
@@ -1,25 +1,56 @@
-const Form = () => (
-  <form className="flex flex-col gap-5 items-start justify-start">
-    <label htmlFor="title">
-      <input
-        type="text"
-        id="title"
-        placeholder="title"
-        className="border-2 border-black"
-      />
-    </label>
-    <label htmlFor="author">
-      <input
-        type="text"
-        id="author"
-        placeholder="author"
-        className="border-2 border-black "
-      />
-    </label>
-    <button type="submit" className="border-2 border-black px-2 py-1">
-      submit
-    </button>
-  </form>
-);
-
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { addBook } from "../redux/books/books";
+import store from "../redux/configureStore";
+function Form() {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState("");
+  const [author, setAutor] = useState("");
```

Comment:
instead of using two separate states, why not create a form object with the two as properties.. this is more scalable and maintainable.

Reply:
All changes made as recommend